### ISSUE TITLE
NullPointerException on Layout.getHeight() while the session replay feature is enabled

### ISSUE THREAD

**Issue by iamironz** [getsentry/sentry-java#3742]
### Integration

sentry-android

### Build System

Gradle

### AGP Version

8.3.1

### Proguard

Disabled

### Version

7.13.0

### Steps to Reproduce

Hey!
Today, I discovered that if I run UI tests on emulators with an enabled session replay feature, some random UI tests fail internally within the Sentry SDK with a fatal NPE on the `Layout.getHeight()` function. 

The issue randomly occurs on the emulator while running the UI tests pipeline with the following configuration:

Model: sdk_gphone_x86_64
OS: Android 11
Screen Resolution: 480x752
Screen DPI: 160
Screen Width Pixels: 480

The stack trace is under the "Actual Result" section.

### Expected Result

Errors related to the internal Android Sentry SDK logic should not affect the app process, especially side effects such as fatal crashes. If any internal SDK exceptions occur, then the appropriate subset of data should not be captured, but the rest of the functionality should remain operational. 

### Actual Result

```txt
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.text.Layout.getHeight()' on a null object reference
	at android.widget.TextView.getVerticalOffset(TextView.java:7369)
	at android.widget.TextView.getTotalPaddingTop(TextView.java:2800)
	at io.sentry.android.replay.viewhierarchy.ViewHierarchyNode$Companion.fromView(ViewHierarchyNode.kt:248)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:265)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.traverse(ScreenshotRecorder.kt:267)
	at io.sentry.android.replay.ScreenshotRecorder.capture$lambda$4$lambda$3(ScreenshotRecorder.kt:123)
	at io.sentry.android.replay.ScreenshotRecorder.$r8$lambda$IWVuUa2tRHJ6zN18kvecvFYJidE(Unknown Source:0)
	at io.sentry.android.replay.ScreenshotRecorder$$ExternalSyntheticLambda2.onPixelCopyFinished(D8$$SyntheticClass:0)
	at android.view.PixelCopy$1.run(PixelCopy.java:191)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at androidx.test.espresso.base.Interrogator.loopAndInterrogate(Interrogator.java:156)
	at androidx.test.espresso.base.UiControllerImpl.loopUntil(UiControllerImpl.java:510)
	at androidx.test.espresso.base.UiControllerImpl.loopMainThreadUntilIdle(UiControllerImpl.java:433)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:329)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:319)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by romtsn** [comment]
hey, I believe this issue was fixed in 7.14.0, specifically [this PR](https://github.com/getsentry/sentry-java/pull/3628/files#diff-4a4832d3579c04fc609fcd22a0fa97007b48b195108b497b2d50056d6fd41e03R93-R103). Could you bump the sdk version and give it another try? Thanks for reporting though!

**Comment by iamironz** [comment]
@romtsn It worked out. 
Thank you for pointing out the latest version of the library :) 

However, during the try attempt, I found out that the latest Sentry AGP `io.sentry.android.gradle` of version `4.11.0` adds the `sentry-android` dependency with version `7.13.0` but not `7.14.0`. As I see, the PR was merged on the same day as the latest release of SDK. 
Is this expected behavior?

In the source code repository, where the actual plugin lives, there is a bit of obscure logic for fetching versions, and I have only found net-exactly-pinned versions that point to the `VERSION_APP_START` constant, which is `7, 1, 0`.
